skipruntime-ts/examples/database-server.ts

```diff
@@ -41,7 +41,7 @@ app.get("/users", (_req, res) => {
   service
     .getStreamUUID("users")
     .then((uuid) => {
-      res.redirect(301, `http://localhost:8080/v1/streams/${uuid}`);
+      res.redirect(301, service.streamURL(uuid));
```

Comment:
In a real world scenario, the `streamURL()` itself will not be exposed as is to the outside world (usually some URL rewriting will take place in the reverse proxy, for instance mapping `http://mywebsite.com/api/sse/{uuid}` to `http://192.168.42.37:8081/v1/streams/{uuid}`. Redirecting to `streamURL()` will issue an HTTP redirect to the latter, which may not be accessible from the client.

Reply:
To check my understanding: So you mean that when people write their client-facing server code, such as this database-server.ts example, then they will want to send redirect URLs to the client code that will then be rewritten by the reverse proxy to the actual URL of the reactive skip service? So it is the reverse proxy configuration that will need to talk about the actual URL of the stream. In which case encapsulating these details just can't work.

Reply:
> So you mean that when people write their client-facing server code, such as this database-server.ts example, then they will want to send redirect URLs to the client code that will then be rewritten by the reverse proxy to the actual URL of the reactive skip service?

That's one option. Basically, the reactive service itself does not know its own public url, which makes it unable to produce public URLs for streams.